staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/errors.go

```diff
@@ -47,9 +47,9 @@ func Forbidden(ctx request.Context, attributes authorizer.Attributes, w http.Res
 
 	var errMsg string
 	if len(reason) == 0 {
-		errMsg = fmt.Sprintf("%s", msg)
+		errMsg = fmt.Sprintf("%s.", msg)
 	} else {
-		errMsg = fmt.Sprintf("%s: %q", msg, reason)
+		errMsg = fmt.Sprintf("%s: %s", msg, reason)
```

Comment:
I also would consider moving the reason into the details field... the ugliest part of the reported issue was that the reason returned from RBAC was incredibly similar to the generic forbidden message constructed from the attributes

Reply:
> I also would consider moving the reason into the details field.

@liggitt This would bring in an api change that affecting other methods. Do we really want to do this?